src/FastRoute/src/DBWrapper.cpp

```diff
@@ -1351,6 +1351,8 @@ void DBWrapper::createMapForDbNets() {
 void DBWrapper::fixAntennas(std::string antennaCellName,
                             std::string antennaPinName)
 {
+  _chip = _db->getChip();
```

Comment:
pointless to even have a dbChip. makes more sense to have a dbBlock.
there are no useful functions on a dbChip